index.js

```diff
@@ -189,7 +189,8 @@ S3Storage.prototype._handleFile = function (req, file, cb) {
         serverSideEncryption: opts.serverSideEncryption,
         metadata: opts.metadata,
         location: result.Location,
-        etag: result.ETag
+        etag: result.ETag,
+        versionId: result.VersionId,
```

Comment:
Although I personally like trailing commas, I like `standard` even more. I believe that the tests are failing due to this trailing comma...